src/hub.html

```diff
@@ -342,6 +343,7 @@
                 class="camera"
                 camera
                 personal-space-bubble="radius: 0.4;"
+                rotation
```

Comment:
Yikes. This is not the first time we've been burned by this. It took us a long time to correct it for the case of network-instantiated objects where we observed the RECEIVERS of an instantiation setting their euler order as YZX while the SENDER's euler was XYZ. 
I saw your message about how this regressed when I changed `pitch-yaw-rotator` so this fix is good.

Reply:
yep, on the `pinned-persistence` branch i have a custom component to just correct this instead of using `rotation`